src/frontend/check_annotation.nit

```diff
@@ -111,6 +111,9 @@ before_all
 after
 after_all
 example
+
+expects
```

Comment:
I found the use of `expects` more judicious seen that in the next version of c ++ (20) the contract will be define with the keywords `expects` and `ensures`. So I think it would be interested to  keep the same principle for the future. But I'm not really fixed on this issue if all users are more for the use of `require` it’s good to me.